_layouts/guides.html

```diff
@@ -16,6 +16,11 @@
     <div class="docslink">
       <a class="returnlink" href="{{site.baseurl}}/{% if versioned_page %}version/{{docversion}}/guides/{% else %}guides/{% endif %}"> Back to Guides</a>
     </div>
+    {% if docversion == 'latest' %}
+    <div class="docslink">
+      <a class="editlink" href="https://github.com/quarkusio/quarkusio.github.io/edit/develop/{{ page.path }}">Edit this Page</a>
```

Comment:
Hhmm, I think this should rather point to https://github.com/quarkusio/quarkus/edit/main/docs/src/main/asciidoc/some-guide.adoc, shouldn't it?

Example:

https://github.com/quarkusio/quarkus/edit/main/docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc

rather than

https://github.com/quarkusio/quarkusio.github.io/edit/develop/_guides/telemetry-micrometer-tutorial.adoc

Hence, I'd need to strip off "_guides" from the `{{ page.path }}`.

Reply:
Done